
<!-- README.md is generated from README.Rmd. Please edit that file -->

# davizr

<!-- badges: start -->
<!-- badges: end -->

*D*ifferential *A*nalysis *VIS*ualiz*ER*

`davizr` is an R package that makes it easy to import and visualize data
for sequencing data differential analysis such as identifying
differentially expressed genes (DEGs) from RNA-seq data, calling
differentially accessible regions (DARs) from ATAC-seq data, etc.

## Installation

You can install the development version of davizr from
[GitHub](https://github.com/) with:

``` r
# install.packages("devtools")
devtools::install_github("trev-f/davizr")
```

## Example

A common task in RNA-seq analysis is simply to construct a matrix of
read counts within genes from a collection of files generated by
`featureCounts`.

``` r
library(davizr)

paths <- file.path(
  testthat::test_path("data"),
  list.files(testthat::test_path("data"), pattern = "sample_[12]_counts\\.txt")
)
read_featureCounts_matrix(paths)
#>           sample_1_sorted sample_2_sorted
#> YAL069W                 0               0
#> YAL068W-A               0               0
#> YAL068C                 0               0
#> YAL067W-A               0               0
#> YAL067C                 0               0
#> YAL066W                 0               0
#> YAL065C                 0               0
#> YAL064W-B               0               0
```
